src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -120,28 +120,9 @@ <h4 ng-if="$ctrl.useV3 === 'true'" class="panel-title  border-bottom-small  visi
             </label>
           </div>
         </div>
-        <div class="col-sm-2" ng-if="$ctrl.hideSpouseDetails">
-          <button type="button" 
-                  id="showSpouseButton"
-                  class="btn btn-link"
-                  style="font-weight:bold"
-                  ng-if="!$ctrl.showSpouseDetails"
-                  ng-click="$ctrl.showSpouseDetails = !$ctrl.showSpouseDetails"
-                  translate='SHOW_SPOUSE'>
```

Comment:
The commit message indicates this button is still here. So the commit message should be updated indicating that we removed the ability to toggle spouse individually.